cypress/support/pages/visitWebSite.cy.js

```diff
@@ -0,0 +1,7 @@
+class MainPage {
```

Comment:
rename files from support/pages, they should contain *cy.js name them homePage.pageObject.js

Reply:
fixed